lib/src/modules/session_replay.dart

```diff
@@ -0,0 +1,86 @@
+// ignore_for_file: avoid_classes_with_only_static_members
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:instabug_flutter/src/generated/session_replay.api.g.dart';
+
+class SessionReplay {
+  static var _host = SessionReplayHostApi();
+
+  /// @nodoc
+  @visibleForTesting
+  // ignore: use_setters_to_change_properties
+  static void $setHostApi(SessionReplayHostApi host) {
+    _host = host;
+  }
+
+  /// Enables or disables Session Replay for your Instabug integration.
+  ///
+  /// Session Replay allows you to capture user interactions with your app
+  /// for analysis and debugging. By default, Session Replay is enabled if
+  /// it is available in your current plan.
+  ///
+  /// Example:
+  ///
+  /// ```dart
+  /// SessionReplay.setEnabled(true); // Enable Session Replay
+  /// SessionReplay.setEnabled(false); // Disable Session Replay
+  /// ```
+  ///
+  /// For more information, see [Enabling/Disabling Session Replay](https://docs.instabug.com/docs/android-session-replay#enablingdisabling-session-replay).
```

Comment:
I think we should remove this reference since it refers to the native Android docs and the Flutter docs aren't ready yet.

Same for the other APIs.